src/main/java/school/lemon/changerequest/java/introduction/hw2/Task3.java

```diff
@@ -3,6 +3,23 @@
 
 public class Task3 {
     public static String comboString(String s1, String s2) {
-        return "";
+        if(s1==null){
+            s1=String.valueOf(s1);
+        }
+        if (s2==null){
+            s2=String.valueOf(s2);
+        }
+        int length1 = s1.length();
+        int length2 = s2.length();
+        if (length1 < length2) {
+            StringBuilder first = new StringBuilder();
+            first.append(s1).append(s2).append(s1);
+            return first.toString();
+        } else {
+            StringBuilder second = new StringBuilder();
+            second.append((s2)).append(s1).append(s2);
+            return second.toString();
+        }
```

Comment:
If `s1==null` it's length is not even `0`, it's `null`, so it will automatically be smaller than `s2.length()`.